src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -70,34 +136,81 @@ default GuildMessageChannel getParentMessageChannel()
         throw new UnsupportedOperationException("Parent of this thread is not a MessageChannel. Parent is type: " + getParentChannel().getType().getId());
     }
 
+    /**
+     * Gets the self member, as a member of this thread.
+     *
+     * <br>If the current account is not a member of this thread, this will return null.
+     *
+     * @return The self member of this thread, null if the current account is not a member of this thread.
+     *
+     * @see #isJoined()
+     */
     @Nullable
     default ThreadMember getSelfThreadMember()
     {
         return getThreadMember(getJDA().getSelfUser());
     }
 
-    //Only have access to this with GUILD_MEMBERS
+
+    /**
+     * Gets a List of all cached {@link ThreadMember members} of this thread.
+     * <br>
+     * <br>The thread owner is not included in this list.
```

Comment:
Unless the bot itself is the thread owner

Reply:
What if the bot is online when the thread is created?
What if the bot is online when the thread is created and it is created from one of the bot's own messages? I think that auto-includes them in the thread. Not sure what that does to this array.